plugins/jobs/web_client/views/JobGraphWidget.js

```diff
@@ -11,7 +11,7 @@ import JobsGraphWidgetTemplate from '../templates/jobsGraphWidget.pug';
 import timingHistoryChartConfig from './timingHistoryChartConfig';
 import timeChartConfig from './timeChartConfig';
 
-export default View.extend({
+const JobGraphWidget = View.extend({
```

Comment:
Yeah, if we can enforce all imports to use the same names for default-exported objects, it will make it much easier to grep for usages in the whole codebase. I'm hoping to enforce this rule further with [`eslint-plugin-import`](https://www.npmjs.com/package/eslint-plugin-import), but it will probably take a little extra work.